admin_ui/src/components/ArrayWidget.vue

```diff
@@ -44,20 +54,28 @@ export default {
     },
     data() {
         return {
-            internalArray: [],
+            internalArray: [] as Array<any>,
         }
     },
     methods: {
-        updateArray($event, index) {
-            this.$set(this.internalArray, index, $event.target.value)
+        updateArray($event: Event, index: number) {
+            Reflect.set(
+                this.internalArray,
+                index,
+                ($event.target as HTMLInputElement).value
+            )
```

Comment:
I didn't know about `Reflect.set`, but looks good!

Reply:
You are right. [Reflect](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Reflect) is new global object introduce in ES6 that allow handle dynamic code.